axelrod/player.py

```diff
@@ -258,6 +258,37 @@ def reset(self):
     def update_history(self, play, coplay):
         self.history.append(play, coplay)
 
+    def check_assumptions(self, attributes, raise_error=True):
+        """
+        Compares the player assumptions to a dictionary of attributes.
+        Generates a warning or error if an assumption is not fulfilled.
+
+        Parameters:
+        -----------
+        attributes: dict
```

Comment:
Can we call this something less generic than `attributes`? Maybe `game_assumptions` or maybe just `assumptions` (similarly elsewhere)

Reply:
It's just that attributes is a common python term (there's a library called `attrs`).

Reply:
yeah i did think maybe to change it - 'assumptions' doesn't feel right because the Game doesn't assume anything, it either does or doesn't have something. maybe `characteristic`?

Reply:
Yeah maybe `game_characteristics` or `game_requirements` ?

Reply:
done